src/components/icons/components/editor/ModeIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import useIcon, { IconProps } from '../../../../hooks/useIcon';
+
+const ModeIcon = (props: IconProps) => {
```

Comment:
We allready have a pencil icon? It's called the EditIcon

Reply:
My bad, didn't see the edit icon. Removed the Mode icon and added the edit icon so it's visible in Storybook.